tests/functional/test.cli.run.js

```diff
@@ -9,14 +9,16 @@ import {
 const EXPECTED_MESSAGE = 'Fake Firefox binary executed correctly.';
 
 describe('web-ext run', () => {
-  it('should accept: --no-reload --source-dir SRCDIR --firefox FXPATH',
+  it('should accept: --no-reload' +
+     ' --source-dir SRCDIR --firefox FXPATH --jsconsole',
```

Comment:
Oh yeah, I forgot about this test. I think it mostly only covers the logic of dealing with the Firefox remote debugger but it wouldn't hurt to add in this new option, thanks.